<launch>
    
    <node pkg="solution_master" type="detect_tetra_node.py" name="detect_tetra_node">
        <param name="weights" value="/home/rad/yolov5/runs/train/exp/weights/best.pt"/>
        <param name="data" value="/home/rad/master/p/data.yaml"/>
        <param name="input_image_topic" value="/camera/color/image_raw"/>
        <param name="conf_thres" value="0.6"/>
    </node>
</launch>

